### DISCUSSION TITLE
3D Rendering of Controller Housing

### DISCUSSION THREAD

**Discussion by spifftek70** [tagyoureit/nodejs-poolController#1054] [General]
I'm attempting to make my Controller Housing more organized and 'pretty', like some others I've seen posted in a few discussions.  To help plan it out, I've created a 3D rendering to help figure out how to layout the components.  I didn't bother rendering cables or wires as that would take soooo much more work.

Here's my [rendering](https://collaborate.shapr3d.com/v/jwS33AITi9jNqbSXuUl_N) so far.  Tell me what you think?  I'm using Shapr3D to create it

**Comment by tagyoureit** [comment]
Nice rendering.  One thing that's very nice is to have circuit breakers in the box.  But otherwise, looks good so far.

**Comment by spifftek70** [comment]
Looking at my rendering and setup, what circuit breakers (make/model) would you recommend?  I was looking at this [one for the 24VDC](https://www.amazon.com/dp/B09NDBRDDP?ref=ppx_yo2ov_dt_b_fed_asin_title), But unsure about the 120V

**Reply by tagyoureit** [reply]
See this thread.  https://github.com/tagyoureit/nodejs-poolController/discussions/515#discussion-4190995

I don't have a personal recommendation, but others like Eaton.

**Reply by spifftek70** [reply]
Thanks... Added to the rendering and on order from Amazon

**Reply by spifftek70** [reply]
@tagyoureit  - Take a look at the [Rendering](https://collaborate.shapr3d.com/v/jwS33AITi9jNqbSXuUl_N) now.  Much more detailed

Had to 3D print a custom fan mount for the container and another one for the Sonoff relay.
